src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs

```diff
@@ -9,6 +9,8 @@
   (and (every? #(or (= "Succeeded" (% "status")) (= "Success" (% "status"))) (submission "workflows"))
     (zero? (count (submission "notstarted")))))
 
+(defn any-running? [submission]
+      (some #(or (= "Running" (% "status")) (= "Submitted" (% "status"))) (submission "workflows")))
```

Comment:
You might consider changing this (and `all-success?`) to use the sets defined below.  I think it would be `(some #(contains? wf-running-statuses) (submission "workflows")))`